src/collections/_documentation/performance-monitoring/performance/transaction-summary.md

```diff
@@ -32,10 +32,12 @@ On initial load, the table displays the slowest occurrences of the transaction a
 - Fastest Transactions
 - Recent Transactions
 
-The table also updates dynamically if you change any of the selections in the global header or when you drill in on a latency segment (applicable when viewing the Latency Histogram).
+The table also updates dynamically if you change any of the selections in the global header or when you drill in on a latency segment (applicable when viewing the Latency Histogram). Results are exportable with the "Export All" button. The choice of which kinds of data to display is configurable, and you can change them by clicking "Edit Columns" at the top right of the table. Note that adding or removing any value-based columns may affect the way that Sentry groups the results.
 
 When viewing transactions, you may want to create more curated views. Click "Open in Discover" above the table to create a custom query to investigate further. For more details, see the full documentation for the Discover [Query Builder](/performance-monitoring/discover-queries/query-builder/).
 
+_Note_: Currently, only transaction data - the transaction name and any attributes the transaction inherits from its root span - is searchable. Data contained in spans other than the root span is not indexed and therefore cannot be searched.
```

Comment:
I don't think this is relevant since there isn't a Search Bar here nor does it talk about spans at the Summary level.

Reply:
Is there a better place to put this? Maybe in the Event Detail page?